homeassistant/loader.py

```diff
@@ -1065,7 +1065,11 @@ def _load_platforms(self, platform_names: Iterable[str]) -> dict[str, ModuleType
 
     async def async_get_platform(self, platform_name: str) -> ModuleType:
         """Return a platform for an integration."""
-        platforms = await self.async_get_platforms([platform_name])
+        # Fast path for a single platform when its already
```

Comment:
when it's already

Reply:
https://github.com/home-assistant/core/pull/113948